src/freenet/config/ConfigCallback.java

```diff
@@ -3,7 +3,54 @@
  * http://www.gnu.org/ for further details of the GPL. */
 package freenet.config;
 
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
 public abstract class ConfigCallback<T> {
+
+	/**
+	 * Create a config callback from lambdas. Needs to be specialized
+	 * in Subtypes of ConfigCallback to match the nodeConfig.register API.
+	 *
+	 * @param set accepts the new value and return an Exception or null (if no exception)
+	 */
+	public static <T> ConfigCallback<T> from(Supplier<T> get, Function<T, Exception> set) {
+		return new ConfigCallback<T>() {
```

Comment:
Did somebody forget to write a test for this? 🤣

Reply:
Yes. That was „I can do this quickly before going to bed“ :sweat: